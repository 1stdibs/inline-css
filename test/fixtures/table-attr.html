<html>
<head>
</head>
<body>
    <table>
        <tr>
            <td>
                <table border="0">
                    <tr>
                        <td>
                            empty
                        </td>
                    </tr>
                </table>
            </td>
        </tr>
    </table>
    <table border="1" cellpadding="2" cellspacing="3">
        <tr>
            <td>
                empty
            </td>
        </tr>
    </table>
</body>
</html>
