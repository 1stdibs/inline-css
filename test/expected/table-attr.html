<html>
<head>
</head>
<body>
    <table border="0" cellpadding="0" cellspacing="0">
        <tr>
            <td>
                <table border="0" cellpadding="0" cellspacing="0">
                    <tr>
                        <td>
                            empty
                        </td>
                    </tr>
                </table>
            </td>
        </tr>
    </table>
    <table border="1" cellpadding="2" cellspacing="3">
        <tr>
            <td>
                empty
            </td>
        </tr>
    </table>
</body>
</html>
